lib/wprof/reporters/file_report.rb

```diff
@@ -0,0 +1,63 @@
+module Wprof
+  module Reporters
+    module FileReport
+      def file_report
+        require 'csv'
+        csv_type = WProf::Config.get_value(:csv_type)
+        path = WProf::Config.get_value(:file_path)
+        if csv_type == 'MIX'
+          write_mix_file(path)
+        elsif csv_type == 'SPLIT'
+          write_split_file(path)
+        end
+      end
+
+      def write_mix_file(path)
+        CSV.open("#{path}/wprof.csv", 'ab') do |csv|
+          @data.to_a.each do |elem|
+            csv << elem
+          end
+        end
+      end
+
+      def write_split_file(path)
```

Comment:
Same as above

Reply:
And look as too much split for same think.